components/x-gift-article/src/v2/AdvancedSharingBanner.jsx

```diff
@@ -0,0 +1,10 @@
+import { h } from '@financial-times/x-engine'
+export const AdvancedSharingBanner = ({ enterpriseEnabled, enterpriseRequestAccess }) => {
+	// show banner only if user is b2b and has advanced sharing enabled
+	return enterpriseEnabled && !enterpriseRequestAccess ? (
+		<div>
+			<div className="share-article-dialog__banner-strip" />
+			<h4 className="share-article-dialog__banner-title">Advanced Sharing</h4>
```

Comment:
is `h4` appropriate here? in the current modal the title "Share this article" uses `h2`

Reply:
`h4` was used here because the `font-size` in the [design](https://www.figma.com/file/4AB1PoaYn5JfIjmXx0a3v1/Advanced-Sharing?type=design&node-id=4200%3A21618&mode=dev) is set to `16px`. The closest header tag for that size is [h4](http://zuga.net/articles/html-heading-elements/). The remaining styling is captured in the `.share-article-dialog__banner-title` scss class.